fix(varname): rename session variable

petergrau opened the pull request:
Will avoid duplicate symbols error when used with cordova-plugin-sfauthenticationsession

If you are also using the cordova-plugin-sfauthenticationsession you will get an error for duplicate symbols. It was fixed in another fork but unfortunately that PR never got merged. RasimKanca/cordova-plugin-aswebauthenticationsession@9b12c34

mochini/cordova-plugin-aswebauthenticationsession#2

jwelker110 reviewed (changes requested):
Thank you for taking the time to create the issue and PR, I appreciate it! Do you mind renaming it to _asAuthenticationVC?

Again, thank you for taking the time to improve the plugin.

Commit:
fix(varname): rename session variable

Will avoid duplicate symbols error
when used with cordova-plugin-sfauthenticationsession

petergrau commented:
@jwelker110 not at all - pushed the update...

jwelker110 reviewed (approved):
Awesome! Thank you!

petergrau commented:
@jwelker110 do you actually plan to publish your plugin to npm? that would be quite handy because then we don't nee to include the repo directly...
